python/src/trezorlib/protobuf.py

```diff
@@ -188,7 +188,6 @@ def __init__(cls, name: str, bases: tuple, d: dict) -> None:
 
 class MessageType(metaclass=_MessageTypeMeta):
     MESSAGE_WIRE_TYPE: Optional[int] = None
-    UNSTABLE: bool = False
```

Comment:
why?

Reply:
ah because the mako does not generate it anyway

Reply:
It looks unused to me. I think it's a relict of some no longer existing logic.

Reply:
I think it ceased to be used in 266955ba65e419943f778019660517352630754f or somewhere around that commit.